src/WellIO.jl

```diff
@@ -1,5 +1,20 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+function load(filepath::String)
```

Comment:
If you check the GslibIO.jl package I mentioned, you will see that the signature of the function is not a `String` but a `File` from FileIO. Can you please refactor the code so that it follows the FileIO conventions?